src/utilities/useAxios.ts

```diff
@@ -0,0 +1,54 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
+import { useCallback } from "react";
+
+export type SendRequestProps = {
+  endpoint: string;
+  method?: "GET" | "POST" | "PUT" | "DELETE";
+  headers?: { [key: string]: string };
+  body?: any;
+  params?: any;
+};
+
+const useAxios = () => {
```

Comment:
Yes I like the idea of having single parameters such as endpoint and config. I'd like to keep the parameters within the function itself as passing them with the hook will exclusively work on single endpoint components and we have a few components with multiple endpoints. We could initialize the hook multiple times, but I am not a fan of that solution as it feels like unneccessary duplication

Reply:
I removed the `SendRequestProps` type and passed the properties directly to sendRequest. This is a short-term solution as discussed offline, as the long-term fix would be to pass the parameters from the hook rather than the function. 

This means we will occasionally have multiple instances of the `useAxios` hook in components, where each instance has its own `sendRequest` function, along with its respective `data, loading, error` states. We will open up a new ticket to get the long-term fix sorted.